### ISSUE TITLE
Procedural entry point could not be located in the dynamic library

### ISSUE THREAD

**Issue by evanerichards** [LykosAI/StabilityMatrix#963]
I thought I had fixed this problem. I guess I thought that I had installed pytorch 2.5 but it seems that comfyUI or Stability Matrix is doing this automatically, because after todays comfyUI update that problem is back.

Reverting to pytorch version: 2.4.1 seemed to fix the problem

_Originally posted by @evanerichards in https://github.com/LykosAI/StabilityMatrix/issues/958#issuecomment-2427056619_

**Comment by mohnjiles** [comment]
Yes, we follow ComfyUI's recommendation of installing the latest PyTorch version. It typically results in faster generations. If you need to use xformers you should be able to upgrade to `0.0.28.post2` for Torch 2.5.0 support.

If you want to stop us from upgrading torch on Comfy updates, you can use the Python Dependencies Override in the 3-dots menu like so:
![image](https://github.com/user-attachments/assets/5831e80f-9f28-4aed-b5c6-0d2797c610ef)
![image](https://github.com/user-attachments/assets/16ac2e25-92b1-4639-869f-90dcad84cba5)

**Comment by evanerichards** [comment]
Oh man. This reply is exactly what I was looking for. Thank you!!
